### ISSUE TITLE
github-add-comment with multiline comment failing with: SyntaxError: EOL while scanning string literal

### ISSUE THREAD

**Issue by r0bj** [tektoncd/catalog#731]
# Expected Behavior

github-add-comment with multiline comment works and adds comment to github.

# Actual Behavior

github-add-comment with multiline comment failing with:
```
SyntaxError: EOL while scanning string literal
```

# Steps to Reproduce the Problem

Use example from task catalog readme page https://github.com/tektoncd/catalog/tree/main/task/github-add-comment/0.4:
```
---
apiVersion: tekton.dev/v1beta1
kind: TaskRun
metadata:
  labels:
    tekton.dev/task: github-add-comment
  name: github-add-comment-to-pr-22
spec:
  taskRef:
    kind: Task
    name: github-add-comment
  params:
    - name: REQUEST_URL
      value: https://github.com/chmouel/scratchpad/pull/46
    - name: COMMENT_OR_FILE
      value: |
        The cat went here and there
        And the moon spun round like a top,
        And the nearest kin of the moon,
        The creeping cat, looked up.
        Black Minnaloushe stared at the moon,
        For, wander and wail as he would,
        The pure cold light in the sky
        Troubled his animal blood.
```

```
  File "/tekton/scripts/script-0-j2plw", line 16
    commentParamValue = "The cat went here and there
                                                   ^
SyntaxError: EOL while scanning string literal
```

# Additional Info

github-add-comment in version 0.2 works fine with multiline comments.

**Comment by chmouel** [comment]
I have sent a fix on #732 , i haven't had the chance to test it, it would be great to let us know if it works if you can test that change

**Comment by r0bj** [comment]
@chmouel I can confirm that your fix works fine, thanks!
